### PR TITLE
For Feb 2024 Collaboration Meeting

### PR DESCRIPTION
**Pull request by JoanneBogart**
Updates needed for Feb 2024 Virtual Meeting

### CODE REVIEW

**Comment by JoanneBogart** [comment]
@EiffL and @jchiang87 
One review ought to be sufficient, whoever can get to it more quickly.   Thanks!

File: index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 <head>
   <meta charset="UTF-8">
-  <title>DESC registration form</title>
+  <title>DESC registration form (under development)</title>
```
Comment (sethdigel) [inline index.html]: '(under development)' can be removed

**Review by sethdigel — APPROVED** [review]
The changes all seem clear and I checked the links on the page.  Also, I verified that the registration form works.

File: index.html (hunk 1 above)
Comment (JoanneBogart) [reply index.html]: Thank you, Seth.

**Pull request merged**